### ISSUE TITLE
Replace dependency list with @thombruce/tnt

### ISSUE THREAD

**Issue by thombruce** [thombruce/nuxt-tnt#1]
Remove dependencies:

```
    "@tailwindcss/forms": "^0.3.3",
    "daisyui": "^1.11.1",
    "lodash": "^4.17.21",
    "luxon": "^2.0.2",
    "postcss": "^8.3.0",
    "pug": "^3.0.2",
```

Replace with `@thombruce/tnt`.

---

Also consider migrating the following to `@thombruce/tnt`:

```
    "@fortawesome/free-solid-svg-icons": "^5.15.4",
    "simple-icons": "^5.10.0",
    "theme-change": "^2.0.2"
```

**Comment by thombruce** [comment]
Note: The Fort Awesome package above is a dependency of a Nuxt-specific version of FontAwesome - this should be examined in detail before commitment to any migration.

The SimpleIcons dependency is made use of by a specific component in the Nuxt module that I make moderate use of for Static sites, but it actually might be very specific to my particular needs. It's brand icons. Font Awesome has many of these too in a separate package. Perhaps it should not be migrated but removed entirely. Perhaps icon sets should be reviewed thoroughly.

Theme Change, now that `@thombruce/tnt` includes Vue Meta, should be trivial to migrate. Less trivial perhaps: The theme-toggle menu.

**Comment by thombruce** [comment]
Closed by 40bc775ab4b19a958743b56d50099a2f0f1d7613
